src/ui/component/filePicker/filePicker.tsx

```diff
@@ -35,24 +41,46 @@ export const FilePicker: FC<FilePickerProps> = ({
     state === 'dragging-over' ? setIsDraggingOver(true) : setIsDraggingOver(false)
   }, [])
 
+  const toFileDTO = useCallback(
+    (file: globalThis.File) => ({
+      name: file.name,
+      fullPath: file.webkitRelativePath,
+      size: file.size,
+      mediaType: file.type,
+      stream: file.stream()
+    }),
+    []
+  )
+
+  const handleFileChange = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>) => {
+      const fileList = event.target.files
+      if (fileList?.length) {
+        const files = Array.from(fileList).map(file => toFileDTO(file))
```

Comment:
You can simplify it :)

```suggestion
const files = Array.from(fileList, toFileDTO)
```